index.js

```diff
@@ -29,34 +29,57 @@ process.title = 'logdna-agent'
+  .option('-t, --tags <tags>', 'Add tags for this host, separate multiple tags by comma'
+  , utils.appender(), [])
+  .option('-l, --list [params]'
+  , 'Show the saved configuration (all unless params specified)'
+  , utils.appender(), false)
+  .option('-u, --unset <params>'
+  , 'Clear some saved configurations (use "all" to unset all except key)'
+  , utils.appender(), [])
+  .option('-s, --set [key=value]', 'Set config variables', utils.appender(), false)
   .on('--help', () => {
     console.log('  Examples:')
     console.log()
     console.log('    $ logdna-agent --key YOUR_INGESTION_KEY')
     console.log('    $ logdna-agent -d /home/ec2-user/logs')
-    console.log('    $ logdna-agent -d /home/ec2-user/logs -d /path/to/another/log_dir  # multiple logdirs in 1 go')
-    console.log('    $ logdna-agent -d "/var/log/*.txt"                                 # supports glob patterns')
-    console.log('    $ logdna-agent -d "/var/log/**/myapp.log"                          # myapp.log in any subfolder')
+    console.log('    $ logdna-agent -d /home/ec2-user/logs -d /path/to/another/log_dir  '
+      + '# multiple logdirs in 1 go')
+    console.log('    $ logdna-agent -d "/var/log/*.txt"                                 '
+      + '# supports glob patterns')
+    console.log('    $ logdna-agent -d "/var/log/**/myapp.log"                          '
+      + '# myapp.log in any subfolder')
     console.log('    $ logdna-agent -f /usr/local/nginx/logs/access.log')
-    console.log('    $ logdna-agent -f /usr/local/nginx/logs/access.log -f /usr/local/nginx/logs/error.log')
-    console.log('    $ logdna-agent -t production                                       # tags')
+    console.log('    $ logdna-agent -f /usr/local/nginx/logs/access.log '
+      + '-f /usr/local/nginx/logs/error.log')
+    console.log('    $ logdna-agent -t production                                       '
+      + '# tags')
     console.log('    $ logdna-agent -t staging,2ndtag')
-    console.log('    $ logdna-agent -l tags,key,logdir                                  # show specific config parameters')
-    console.log('    $ logdna-agent -l                                                  # show all')
-    console.log('    $ logdna-agent -u tags,logdir                                      # unset specific entries from config')
-    console.log('    $ logdna-agent -u all                                              # unset all except LogDNA API Key')
+    console.log('    $ logdna-agent -l tags,key,logdir                                  '
+      + '# show specific config parameters')
+    console.log('    $ logdna-agent -l                                                  '
+      + '# show all')
+    console.log('    $ logdna-agent -u tags,logdir                                      '
+      + '# unset specific entries from config')
+    console.log('    $ logdna-agent -u all                                              '
+      + '# unset all except LogDNA API Key')
```

Comment:
This is a bit hard to read and will be hard to maintain. You could use a single multi-line string template
```javascript
`
  this can contain 
  text on multiple lines
`.trim()
```

that can be a little hard to keep spacing, so the other option would be to put the help text formatted cleanly in a separate file and just console.log the contents of the file.

Reply:
The issue I had was that this is a single line text and eslint had an issue with the character count. If it with a string template, it will print out the new line as well. That's why I didn't go with it at first.

Reply:
I know i can escape the new line but the formatting is off. I can adjust the other ones to match